Doc/library/math.rst

```diff
@@ -220,12 +223,15 @@ Power and logarithmic functions
 
 .. function:: exp(x)
 
-   Return e raised to the power *x*, where e = 2.718281... is the base
-   of natural logarithms.
+   Return *e* raised to the power *x*, where *e* = 2.718281... is the base
+   of natural logarithms.  This is usually more accurate than ``math.e ** x``
+   or ``pow(math.e, x)``.
+
 
 .. function:: expm1(x)
 
-   Return ``e**x - 1``.  For small floats *x*, the subtraction in ``exp(x) - 1``
+   Return *e* raised to the power *x* minus 1, where *e* is the base of natural
```

Comment:
This is ambiguous: it could be read as "e raised to the power (x minus 1)". I think an extra comma would remove the ambiguity: "Return *e* raised to the power *x*, minus 1. Here *e* is the base ..."